### ISSUE TITLE
Regression: `common:testDebugUnitTest` task no longer exists

### ISSUE THREAD

**Issue by NachoSoto** [RevenueCat/purchases-android#490]
I investigated [my report a few days ago](https://github.com/RevenueCat/purchases-android/pull/444#pullrequestreview-883581383), where right-clicking a class or test to run it no longer works, and found the source of the problem.
Prior to #444 these were the tasks available:
```
common:test - Run unit tests for all variants.
common:testDebugUnitTest - Run unit tests for the debug build.
common:testReleaseUnitTest - Run unit tests for the release build.
```

After the PR, `testDebugUnitTest` no longer exists:

```
common:test - Run unit tests for all variants.
common:testLatestDependenciesDebugUnitTest - Run unit tests for the latestDependenciesDebug build.
common:testLatestDependenciesReleaseUnitTest - Run unit tests for the latestDependenciesRelease build.
common:testUnityIAPDebugUnitTest - Run unit tests for the unityIAPDebug build.
common:testUnityIAPReleaseUnitTest - Run unit tests for the unityIAPRelease build.
```

This is a problem because Android Studio defaults to `:common:testDebugUnitTest` when creating a run configuration, which is why manually running a test no longer works unless you change that to `:common:test`.

**Comment by NachoSoto** [comment]
Possibly related, but it looks like debugging tests doesn't work either, breakpoints are never hit.

**Comment by vegaro** [comment]
@NachoSoto I can run tests if doing right click in a class and even on a module in the latest `main`. I just did right click in the `google` module and ran the 76 tests inside successfully. I have the `latestDependenciesDebug` variant selected in the Build Variants window. 

Sometimes Android Studio can't find tests but I've found that after a Invalidate caches and restart of Android Studio or a restart of my computer, they start working again. Let me know if I can help in any way.
